core/src/main/java/com/adobe/aem/commons/assetshare/components/predicates/impl/options/SortOptionItem.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Asset Share Commons
+ *
+ * Copyright (C) 2017 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.adobe.aem.commons.assetshare.components.predicates.impl.options;
+
+public class SortOptionItem {
```

Comment:
Can this implement/extend OptionItem? I mainly ask to break the Sort Model's public API less. I assume there's a class that takes care of a bit of this already (since caseSensitive is the only new thing)

Reply:
Changed